boilerplate/js/package.json

```diff
@@ -7,8 +7,13 @@
     "webpack": "^4.26.0",
     "webpack-cli": "^3.0.7"
   },
+  "devDependencies": {
+    "prettier": "2.0.2"
+  },
   "scripts": {
     "dev": "webpack --mode development --watch",
-    "build": "webpack --mode production"
+    "build": "webpack --mode production",
+    "format": "prettier --write src",
```

Comment:
Why don't we allow Prettier to format the entire extension, including Less, YML, JSON, etc? It'd help to maintain styles even more, especially as there is currently nothing in place for these.

Reply:
I don't think the JS portion of the extension should be responsible for formatting anything outside of the JS portion of the extension. We could do this, but then we should move the formatting task to `composer.json`.

Reply:
That would probably be a good idea.

Might it be worth using Prettier to format the PHP, too? There's an [officially maintained plugin for PHP formatting](https://github.com/prettier/plugin-php), and that means we wouldn't be combined different tools (like StyleCI) with Prettier.

Reply:
If we want to do that, I think we should consider it as a separate issue from the stuff here: my goal with this PR is to bring the template up to date with recent core changes.